Demo/Test/MeasuresTest.cs

```diff
@@ -88,5 +92,72 @@ public void BCSEHEDIS2022_Numerator_FromElm()
             Assert.IsFalse((bool?)numerator);
         }
 
+        [UsedImplicitly]
+        public static AssemblyLoadContext LoadResources(
```

Comment:
I think this high-level API should remain in our SDK. Or is there another way for "end users" to load Libraries and execute a library?  Otherwise, everyone has to rewrite this boilerplate in their code, right?

Reply:
I think we should work on the internals until we happy, then we can move back the public API back?